frontend/src/app/shared/components/datepicker/datepicker.modal.service.ts

```diff
@@ -209,15 +209,29 @@ export class DatepickerModalService {
 
     /* eslint-disable no-param-reassign */
```

Comment:
We can now reduce the scope of this disable comment to only the `if` on line 218